### PR TITLE
Rename VirtualMachine and OfflineVirtualMachine

### PR DESCRIPTION
**Pull request by rmohr**
With the rename of VirtualMachine  to VirtualMachine instance the following things need to be changed in the guide:

 - [x] Rename structs: VirtualMachine to VirtualMachineInstence
 - [x] Rename structs: OfflineVirtualMachine to VirtualMachine
 - [x] Go through all texts and ensure consistency
 - [x] Rename abbreviations and variants (VM, vm, ovm, VirtualMachine's, ...)

The rename itself is in https://github.com/kubevirt/kubevirt/pull/1102

### CODE REVIEW

File: workloads/controllers/offline-virtual-machine.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ create the controller instance:
 
 ```bash
 $ kubectl create -f ovm.yaml 
-offlinevirtualmachine "ovm-cirros" created
+virtualmachine "ovm-cirros" created
```
Comment (davidvossel) [inline workloads/controllers/offline-virtual-machine.md]: ovm

File: workloads/controllers/offline-virtual-machine.md [hunk 2]
```diff
@@ -237,29 +237,29 @@ demonstrating how to do it.
 kubectl create -f myofflinevm.yaml
```
Comment (davidvossel) [inline workloads/controllers/offline-virtual-machine.md]: myofflinevm -> myvm

File: workloads/virtual-machines/presets.md [hunk 3]
```diff
@@ -4,13 +4,13 @@
 
 `VirtualMachinePresets` are an extension to general `VirtualMachine` configuration behaving much like `PodPresets` from Kubernetes. When a `VirtualMachine` is created, any applicable `VirtualMachinePresets` will be applied to the existing spec for the `VirtualMachine`. This allows for re-use of common settings that should apply to multiple `VirtualMachines`.
```
Comment (davidvossel) [inline workloads/virtual-machines/presets.md]: VirtualMachineInstancePresets. run a filter with the 's' on it

File: workloads/virtual-machines/presets.md [hunk 4]
```diff
@@ -45,7 +45,7 @@ Once a `VirtualMachinePreset` is successfully applied to a `VirtualMachine`, the
 
 Any valid `Label` can be matched against, but it is suggested that a general rule of thumb is to use os/shortname, e.g. `kubevirt.io/os: rhel7`.
 
-### Updating a VirtualMachinePreset
+### Updating a VirtualMachineInstancePreset
 
 If a `VirtualMachinePreset` is modified, changes will _not_ be applied to existing `VirtualMachines`. This applies to both the `Selector` indicating which `VirtualMachines` should be matched, and also the `Domain` section which lists the settings that should be applied to a `VirtalMachine`.
```
Comment (davidvossel) [inline workloads/virtual-machines/presets.md]: VirtualMachinePreset

**Comment by rmohr** [comment]
@cynepco3hahue @davidvossel should now be in a reasonable shape. More needs to be done once the vm will be more reshaped into the workload direction.

File: workloads/controllers/virtual-machine-replica-set.md [hunk 5]
```diff
@@ -1,66 +1,66 @@
...
-For example, exposing SSH port (22) as a ClusterIP service using virtctl on a VirtualMachineReplicaSet:
+For example, exposing SSH port (22) as a ClusterIP service using virtctl on a VirtualMachineInstanceReplicaSet:
 
 ```bash
-$ virtctl expose vmrs vm-ephemeral --name vmservice --port 27017 --target-port 22
+$ virtctl expose vmirs vmi-ephemeral --name vmiservice --port 27017 --target-port 22
 ```
 
-All service exposure options that apply to a VirtualMachine apply to an VirtualMachineReplicaSet. See [Exposing VirtualMachine](http://www.kubevirt.io/user-guide/#/workloads/virtual-machines/expose-service) for more details.
+All service exposure options that apply to a VirtualMachineInstance apply to an VirtualMachineInstanceReplicaSet. See [Exposing VirtualMachineInstance](http://www.kubevirt.io/user-guide/#/workloads/virtual-machines/expose-service) for more details.
```
Comment (davidvossel) [inline workloads/controllers/virtual-machine-replica-set.md]: one more filter to run. 's/[an Virtual/a Virtual' 

There are a few instances of this left

File: workloads/controllers/virtual-machine.md [hunk 6]
```diff
@@ -1,28 +1,28 @@
...
 It focuses on a 1:1 relationship between the controller instance and a virtual
 machine instance. In many ways it is very similar to a
 [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/)
 with `spec.replica` set to `1`.
 
-## How to use an OfflineVirtualMachine
+## How to use a VirtualMachine
 
-An OfflineVirtualMachine will make sure that a VirtualMachine object with an
+An VirtualMachine will make sure that a VirtualMachineInstance object with an
```
Comment (davidvossel) [inline workloads/controllers/virtual-machine.md]: missed this one

File: workloads/controllers/virtual-machine.md [hunk 7]
```diff
@@ -36,59 +36,59 @@ virtctl stop myvm
...
-kubectl delete virtualmachine myvm
+# Restart the virtual machine (you delete the instance!):
+kubectl delete virtualmachineinstance myvm
 ```
 
 ### Fencing considerations
 
-An OfflineVirtualMachine will never restart or re-create a VirtualMachine until
-the current instance of the VirtualMachine is deleted from the cluster.
+An VirtualMachine will never restart or re-create a VirtualMachineInstance until
```
Comment (davidvossel) [inline workloads/controllers/virtual-machine.md]: and this

File: workloads/controllers/virtual-machine.md (hunk 6 above)
Comment (rmohr) [reply workloads/controllers/virtual-machine.md]: Done

File: workloads/controllers/virtual-machine.md (hunk 7 above)
Comment (rmohr) [reply workloads/controllers/virtual-machine.md]: Done

**Pull request merged**